CONTRIB.md

```diff
@@ -0,0 +1,16 @@
+## Contributing
+
+### Submission and Review Process
+
+1. Submit a PR for all proposed changes
+  1. open PR's that are not ready for review should be labelled that way
+    1. use the "wip" label, or;
+    1. prefix the PR title with "WIP" or "Work In Progress"
+  1. open PR's that are ready for review should be labelled that way
+    1. remove the "wip" label, or "WIP" prefixing
+    1. apply "PTAL" label, and in a comment '@cc' people that you think should be reviewing the PR
```

Comment:
In Mesos there is a concept of a "shepherd" who helps land each patch.  I like this since it avoids the problem of shared responsibility.  We could use the assignee for this.  The assignee should be responsible for ultimately clicking the "merge" or "close" button IMO.